web_app_drawer/static/src/less/variables.less

```diff
@@ -0,0 +1,15 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+// App Drawer / Icons
+
+@app-drawer-icon-size:      5.625em;
+@app-drawer-icon-margin:    0.625em;
+@app-drawer-width:          80%;
+@app-drawer-title-height:   2.9em;
+
+// Navbar
+
+@navbar-height:  2.3em;
+@navbar-padding-vertical: .565em;
+@navbar-padding-horizontal: .565em;
```

Comment:
Not entirely bad, since you can always redefine them later in the stack with your theme, but I'm more in the side of leaving all theming for themes (while providing sane defaults, of course).

So if you think original is too much, you could just use `@navbar-height / 2` instead of redefining it and affecting the rest of the theme.

Anyway, I talked today with @pedrobaeza that I will probably send a little PR over yours when you finish most logic to polish some little design thingies that are just too little to comment, but that summing all of them you get much more polished aspect IMHO.

I don't know if you understand Spanish, but this was my mockup:

![captura de pantalla de 2016-08-25 10-42-45](https://cloud.githubusercontent.com/assets/973709/17975714/0c15c094-6aeb-11e6-88bf-79ce48cd44fc.png)